akka-stream/src/main/scala/akka/stream/scaladsl/FileIO.scala

```diff
@@ -49,7 +50,6 @@ object FileIO {
    * This source is backed by an Actor which will use the dedicated `akka.stream.blocking-io-dispatcher`,
    * unless configured otherwise by using [[ActorAttributes]].
    */
-  def toFile(f: File, append: Boolean = false): Sink[ByteString, Future[IOResult]] =
-    new Sink(new FileSink(f, append, DefaultAttributes.fileSink, sinkShape("FileSink")))
-
+  def toFile(f: File, options: Set[StandardOpenOption] = Set(WRITE, CREATE)): Sink[ByteString, Future[IOResult]] =
```

Comment:
Perhaps add to the documentation what the default parameters are? (There are no @params present in the Scaladoc)